qutebrowser/browser/commands.py

```diff
@@ -1175,6 +1175,35 @@ def tab_move(self, index: typing.Union[str, int] = None, count=None):
         cmdutils.check_overflow(new_idx, 'int')
         self._tabbed_browser.tabBar().moveTab(cur_idx, new_idx)
 
+    @cmdutils.register(instance='command-dispatcher', scope='window',
+                       debug=True)
```

Comment:
I don't think the `debug=True` makes sense here. I guess it'll just show up in the completion, no biggie.